ml/requirements.txt

```diff
@@ -0,0 +1,24 @@
+certifi==2024.7.4
```

Comment:
same as sub-.gitignore - can we merge that with the existing requirements.txt?
As I understand we will use all of that ML related code in the validator itself so it will need same requirements and the global ones